modules/system/console/MixWatch.php

```diff
@@ -115,9 +120,12 @@ public function handleSignal(int $signal): void
         // Cleanup
         $this->removeWebpackConfig(base_path($this->mixJsPath));
 
-        // Exit cleanly at this point, if this was a user termination
-        if ($signal === SIGINT || $signail === SIGQUIT) {
-            exit(0);
+        //Windows fix for signal issue
+        if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN') {
+            // Exit cleanly at this point, if this was a user termination
+            if ($signal === SIGINT || $signail === SIGQUIT) {
+                exit(0);
+            }
```

Comment:
Would you be willing to add support for Window's equivalent API to handle console command cleanup? See https://stackoverflow.com/questions/9523240/php-cli-in-windows-handling-ctrl-c-commands/60519075#60519075

Reply:
I can add the signal handling this week :) 👍

Reply:
@iamyigitkoc are you able to add the cleanup functionality for Windows based on that stackoverflow link?